test/e2e/vsn.js

```diff
@@ -0,0 +1,191 @@
+'use strict'
+
+const assert = require('assert')
+const tapePromise = require('tape-promise').default
+const tape = require('tape')
+const isRoughlyEqual = require('is-roughly-equal')
+
+const {createWhen} = require('./lib/util')
+const createClient = require('../..')
+const vsnProfile = require('../../p/vsn')
+const products = require('../../p/vsn/products')
+const {
```

Comment:
Remove these, AFAICT they're not needed.

Reply:
hmm, without them, the code will not run...

Reply:
But they're not used in the file.

Reply:
I will remove this when merging.